exonum/src/blockchain/mod.rs

```diff
@@ -392,7 +392,9 @@ impl Blockchain {
         mount
     }
 
-    fn api_context(&self) -> ApiContext {
+    // TODO: I need the public key - how can i get it?
+    // is it ok to make this public?
+    pub(crate) fn api_context(&self) -> ApiContext {
```

Comment:
It seems that the method should be public and documented.
And later we may be add possibility to add the custom handlers to the `NodeBulder`.